src/admin/paths/admin@reports@process_summary@bulk.yaml

```diff
@@ -0,0 +1,68 @@
+post:
+  tags:
+    - Reports
+  summary: Fetch reimbursable amount summary before processing reports
```

Comment:
**generate** reimburable amount summary 
fetch sounds like a GET action 

also API path can be `/admin/reports/generate_summary/bulk` IMHO